### DISCUSSION TITLE
Moving Horreum to it's own org?

### DISCUSSION THREAD

**Discussion by johnaohara** [Hyperfoil/Horreum#648] [General]
At present, Horreum is hosted under the Hyperfoil org, co-located with hyperfoil repositories in github and published under `io.hyperfoil.tools`.  Historically this was convenient as there was a tighter integration and we only had to manage one domain, org etc. but currently causes confusion between users and contributors.

As Horreum has grown, I think it makes sense to move Horreum (and related projects) from under the Hyperfoil org and maintain seperate orgs.  I have already registered a new Github organisation: https://github.com/Horreumio (unfortunately, https://github.com/Horreum was taken)

We have already registered horreum.io as a domain name, currently it points to https://horreum.hyperfoil.io/

Moving org would require;

1. Moving Horreum related repositories from https://github.com/Hyperfoil/ to https://github.com/HorreumIO/
2. Renaming published artefacts, e.g. Java modules `io.hyperfoil.tools.horreum` -> io.horreum`
3. Changing maven published artefacts
4. Updating website/docs with changed domain name
5. Branding for Horreum
6. Updating downstream projects that consume Horreum artefacts

**Comment by shivarm** [comment]
@johnaohara when will be we doing it?

**Reply by shivarm** [reply]
@johnaohara any thought?

**Reply by jesperpedersen** [reply]
This is a TODO - it is not right now

**Reply by shivarm** [reply]
okay, so now horreum grown

**Comment by stalep** [comment]
I'm very positive to such a move, but we need to fit it into our prio. I'm thinking it would be good if we could do this before Christmas. We need to look at the implications/work required to move.

**Reply by shivarm** [reply]
after my exam I will give a try.

**Comment by stalep** [comment]
It will require us to test/check releasing under the horreum.io namespace, a new homepage, branding, ++
If you want to start that work, that's great, but I do not think it's something we can prioritize as a team right now.

**Reply by johnaohara** [reply]
+ redirects in maven central (https://mvnrepository.com/artifact/io.hyperfoil.tools/), new organisation in sonatype (https://s01.oss.sonatype.org/content/repositories/releases/)  migrating downstream projects (https://github.com/jenkinsci/horreum-plugin) etc there is quite a bit of work to setup a migration.  Plus we need a logo ;)

**Comment by shivarm** [comment]
tracking https://github.com/Hyperfoil/Horreum/issues/979
